src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_issue_name!(name)
```

Comment:
I prefer the one liner, I see it more legible from my POV. The other way is harder, having an explicit return only in one case and then returning true implicitly only in the other case. I always prefer a boolean expression instead of having flow control structures (**return** and **if**).

Also if you want to make it more clear we can have a method in the tracker and ask like this:
```ruby
!tracker.is_cve? || Regexp.new(/^\d\d\d\d-\d+$/).match(name) 
```